testsuite/unittests/test_gif_cond_exp.sli

```diff
@@ -0,0 +1,112 @@
+/*
+ *  test_gif_cond_exp.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+ /* BeginDocumentation
+Name: testsuite::test_gif_cond_exp - sli script for test of gif_cond_exp with external DC current and spike generators
+
+Synopsis: (test_gif_cond_exp) run
+
+
+Description:
+
+ This testscript checks the correctness of gif_psc_exp neuron model. It injects external DC current and also three spikes to two neurons and measures their spike times. The spike times should match the expected values.
+
+Author:  March 2016, Setareh
+SeeAlso: gif_cond_exp, testsuite::test_gif_cond_exp_multisynapse
+*/
```

Comment:
Thanks for your suggestions. Probably my main problem here is that I am not familiar with the features of SLI. I usually write SLI scripts based on the accumulator language I have learnt in my undergraduate studies.
I have made the function and run it for each test. Besides, I added two tests with wrong parameters (one with negative lambda and another with unequal size of sfa arrays).